src/App.vue

```diff
@@ -223,6 +225,13 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
   /** Whether the token refresh service is initialized. */
   private tokenService: boolean = false
 
+  /** Data for fee summary calculator. */
```

Comment:
Component not calculator.
(it may be a calculator, but in this file we don't care; to us it's just a component)
(the idea is to only know what we need to know)

Reply:
You are correct! Will update :)